jump-gdx/src/main/java/com/bitdecay/jump/gdx/level/RenderableLevelObject.java

```diff
@@ -8,4 +8,13 @@
  */
 public abstract class RenderableLevelObject extends LevelObject {
     public abstract TextureRegion texture();
+
+    public RenderableLevelObject newInstance(){
```

Comment:
Is this here so that subclasses can override it in case we need special construction of a given object?